<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mycreate</class>
 <widget class="QDialog" name="mycreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加学生</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="placeholderText">
        <string>请输入姓名（必填）</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id">
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="placeholderText">
        <string>请输入至少11位学号（必填）</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0,3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_male">
       <property name="text">
        <string>男</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_female">
       <property name="text">
        <string>女</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sexGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电话</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_pnum">
       <property name="placeholderText">
        <string>请输入11位电话号码</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>学校</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_school">
       <item>
        <property name="text">
         <string>请选择学校</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>华南师范大学（石牌校区）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>华南师范大学（大学城校区）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>华南师范大学（南海校区）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其它</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>学院</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_yx">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>请选择学院</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>专业</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_zy">
       <item>
        <property name="text">
         <string>请选择专业</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="te_other"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,3,3,3,0">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>mycreate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="sexGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
